.gitignore

```diff
@@ -102,3 +107,8 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+lookout/core/tests/server
```

Comment:
nit: I guess this comes from a different repo

Reply:
Whoops